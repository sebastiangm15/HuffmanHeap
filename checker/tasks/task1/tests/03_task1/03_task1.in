6
8 ZSAT
8 ASAT
8 BSAT
8 CSAT
10 DSAT
12 ESAT
