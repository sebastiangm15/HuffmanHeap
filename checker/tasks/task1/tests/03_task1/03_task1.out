54-DSATESATASATBSATCSATZSAT 
22-DSATESAT 32-ASATBSATCSATZSAT 
10-DSAT 12-ESAT 16-ASATBSAT 16-CSATZSAT 
8-ASAT 8-BSAT 8-CSAT 8-ZSAT 
